.gitlab-ci.yml

```diff
@@ -127,6 +127,30 @@ build_monit_image:
         - src/python/TaskWorker/__init__.py
       policy: pull
 
+build_crabtwfilebeat_image:
```

Comment:
(optional) could you please make this job and also build monit as optional or release-only-pipeline job?
Simply for minimize pipeline time. You never need this in your normal crab development.
And it is small, independent enough to do a manual build easily.

Reply:
aaaah good point, i should have fixed this, thanks for the comment man